ftest.sh

```diff
@@ -64,9 +64,48 @@ restore_dist_files() {
 
 }
 
+cleanup() {
+    restore_dist_files "${yaml_files[@]}"
+    i=5
+    while [ $i -gt 0 ]; do
+        pdsh -l ${REMOTE_ACCT:-jenkins} -R ssh -S -w "$(IFS=','; echo "${nodes[*]}")" "set -x
```

Comment:
(style)  line over 80 characters
(lint) Double quote to prevent globbing and word splitting. [SC2086]